ImageProcessingRequest_api

```diff
@@ -0,0 +1,86 @@
+v1.ImageProcessingRequest
+
+This is an object that describes request to the ChRIS system to be able to process an image into a desired format
+
+Fields:
+    - imageprocessingrequestid: an autogenerated id that corresponds to the submitted job
+    - patientid: The id of the patient
+    - jobtype: The type of image processing job you want to run
+    - jobstatus: The status of the job (e.g. 'Pending', 'Running', 'Failed', 'Success')
```

Comment:
I was considering the case where you might want to provide additional parameters to the processor.  Say you have an processor that performs some operation.  The operation has several different modifications that can be made depending on how fast you need the data to be processed, and the quality of the input data (suppose we have 6 different options).  You probably don't want to have to produce a separate Docker image for every combination of options.